src/MySqlite.php

```diff
@@ -186,6 +186,15 @@ public static function concat(array $columns, ?string $as = null): Expression
         );
     }
 
+    public static function groupConcat(array $columns, string $separator = ', ', ?string $as = null): Expression
+    {
+        return self::raw(
+            DB::getDefaultConnection() === 'sqlite'
+                ? 'GROUP_CONCAT(' . implode(', ', $columns) . ', ' . "$separator" . ')' . self::as($as)
```

Comment:
Not quite. That has just made a PHP string interpolation. There must be literal quotation marks around the separator, per: `GROUP_CONCAT(val, ';')`